valohai_yaml/objs/input.py

```diff
@@ -37,13 +44,15 @@ def __init__(
         description: Optional[str] = None,
         keep_directories: KeepDirectoriesValue = False,
         filename: Optional[str] = None,
+        override_mode: Optional[InputOverrideMode] = InputOverrideMode.APPEND,
```

Comment:
`Optional` implies that `None` is a valid `override_mode`. Is that correct?

Reply:
One doesn't have to forcefully define override mode on each input. But as we are already assigning a default value i think Optional can be omitted now.